File: applicatie/src/main/java/com/vdzon/administratie/rest/overzicht/OverzichtResource.java
package com.vdzon.administratie.rest.overzicht;

import com.vdzon.administratie.util.JsonUtil;
import spark.Spark;

import javax.inject.Inject;


public class OverzichtResource {

    @Inject
    public OverzichtResource(OverzichtService overzichtService) {
        Spark.get("/rest/overzicht/pdf/:beginDate/:endDate", overzichtService::getPdf, JsonUtil.json());
    }


}

File: applicatie/src/main/java/com/vdzon/administratie/rest/overzicht/OverzichtResource.kt
package com.vdzon.administratie.rest.overzicht

import com.vdzon.administratie.util.JsonUtil
import spark.Route
import spark.Spark

import javax.inject.Inject


class OverzichtResource
@Inject
constructor(overzichtService: OverzichtService) {
    init {
        Spark.get("/rest/overzicht/pdf/:beginDate/:endDate", Route { req, res -> overzichtService.getPdf(req, res) }, JsonUtil.json())
    }


}
